lib/dynflow/action/suspended.rb

```diff
@@ -9,6 +17,15 @@ def initialize(action)
       @step_id           = action.run_step_id
     end
 
+    def wakeup_info
+      @wakeup_info ||= []
+    end
+
+    def wakeup_info=(info)
+      Type! info, WakeupInfo
+      wakeup_info << info
+    end
+
     def event(event, future = Concurrent::Promises.resolvable_future)
       @world.event execution_plan_id, step_id, event, future
```

Comment:
We now have two ways how to send events to the orchestrator, wich seems very suboptimal :disappointed: